src/classes/RecipeRepository.js

```diff
@@ -23,25 +23,17 @@ class RecipeRepository {
     return recipes;
   }
 
-  findRecipesByName(name) {
-    const foundRecipe = this.recipes.find((recipe) => {
-      return recipe.name === name;
-    })
-    return foundRecipe;
-  }
+  findRecipes(searchTerm) {
```

Comment:
Be careful with consolidating functionality like this. While it could make it easier for search, it could make things harder in other parts of the application. It also runs against the concept of single-responsibility. We may not want to search against all of these things every time. 

This is also a very expensive operation if we only need to search against, say, "tags" on the home page. To avoid this, your search functionality should use both `findRecipesByName` _and_ `findRecipesByIngredient` with a given search term and see if there's a result from either (or both!).

I would encourage you to revert these changes and update your search function to do this, but will leave the decision making to you and then team 🙂